### DISCUSSION TITLE
Save app layout for specific Finder windows (by path)?

### DISCUSSION THREAD

**Discussion by jtelcontar** [rxhanson/RectanglePro-Community#442] [Q&A] [answered]
I recently downloaded Rectangle Pro to try it out since I was wanting to be able to save custom app layouts.

Specifically, I am wanting to arrange a bunch of Finder windows in different shapes/sizes. But not just any Finder windows-- ones at specific paths. So my App layout might be `/Users/jtelcontar/Desktop/AwesomeFolder1` in the top left, 100 x 50 px; `/Users/jtelcontar/Desktop/AwesomeFolder2` in the middle left, 20 x 20 px; `/Users/jtelcontar/Desktop/AwesomeFolder3` in the bottom left, 85 x 100 px (etc).

It looks like I can create an App Layout for Finder from my existing window layout. But it doesn't seem like it tracks that my AwesomeFolders are the ones placed specifically in those positions-- just that I want Finder windows in those positions (I edited one of the saved windows in my app layout and it doesn't seem to indicate the path matters, just that it's a Finder window).

(I haven't actually closed all my windows to verify this-- I didn't want to have to set everything up again cause it's actually more than just the 3 folders in my example haha)

Is this something that is possible to do already? If not, can I officially suggest this as a feature for App Layouts? :)

**Comment by rxhanson** [comment, marked as answer]
Thanks for the feedback. This is not possible currently in Rectangle Pro, but it is on my backlog. I don't have an ETA for it at the moment though.

**Reply by jtelcontar** [reply]
Gotcha! No worries, I'll keep an eye out for future updates. Appreciate the reply!
